### PR TITLE
README.md for azure-test-watcher

### PR DESCRIPTION
**Pull request by JimSuplizio**
Missing readme for azure-test-watcher was causing the verify readme step to fail.

### CODE REVIEW

File: common/azure-test-watcher/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+If you would like to become an active contributor to this project please follow the instructions provided in [Microsoft
+Azure Projects Contribution Guidelines](http://azure.github.io/guidelines.html).
+
+1. Fork it
+1. Create your feature branch (`git checkout -b my-new-feature`)
+1. Commit your changes (`git commit -am 'Add some feature'`)
+1. Push to the branch (`git push origin my-new-feature`)
+1. Create new Pull Request
+
+![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-java%2Fcommon%2Fperf-test-core%2FREADME.png)
```
Comment (g2vinay) [inline common/azure-test-watcher/README.md:43]: this link refers to perf-test-core

Comment (JimSuplizio) [reply common/azure-test-watcher/README.md:43]: *sigh*

**Pull request merged**